.bash_prompt

```diff
@@ -57,6 +57,13 @@ else
   sexy_bash_prompt_symbol_color="" # NORMAL
 fi
 
+# Define the default prompt terminator character '$'
+if [[ "$UID" = 0 ]]; then
```

Comment:
Does a single equals sign work? I see that in the other location we used `==` and was wondering if we should use `-eq`.